docs/schema.graphql

```diff
@@ -506,7 +506,8 @@ type PolyjuiceCreator {
   hash_type: String
   id: Int
   script_args: String
-  tx_hash: HashFull
+  tx_hash: HashFull,
+  created_account: String
```

Comment:
it creates the naming convention. We have `from_account` and `to_account` in the schema and they are both of `Account type`, so the `created_account` is expected to be an `Account` too.

Reply:
It's fixed. Current return 
```
PolyjuiceCreator {
  created_account {
     eth_address
     type
     scirpt_hash
  }
}
```